pkg/api/v1/schema.go

```diff
@@ -693,21 +695,26 @@ type InterfaceBridge struct{}
 
 // ---
 // +k8s:openapi-gen=true
-type InterfaceSlirp struct {
-	// List of ports to be forwarded to the virtual machine.
-	Ports []Port `json:"ports,omitempty"`
-}
+type InterfaceSlirp struct{}
 
 // Port repesents a port to expose from the virtual machine.
 // Default protocol TCP.
-// Default port is PodPort.
+// Port is mandatory.
```

Comment:
It would be slightly less confusing if it would say: "The `port` field is mandatory." because the API field is named "port" (lower case).